tools/mkmapdata/mkmapdata.sh

```diff
@@ -297,6 +297,13 @@ do
       echo $QUERYFIL
    fi
    INGRID[nfile]=`$QUERYFIL`
+   INGRID[1]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.5x0.5_nomask_c191114.nc"
+   INGRID[2]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.25x0.25_nomask_c191114.nc"
+   INGRID[3]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.125x0.125_nomask_c191114.nc"
+   INGRID[4]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_5x5min_nomask_c191114.nc"
+   INGRID[5]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_10x10min_nomask_c191114.nc"
+   INGRID[6]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.9x1.25_nomask_c191114.nc"
+   INGRID[7]="/glade/p/cesm/cseg/inputdata/lnd/clm2/mappingdata/grids/UNSTRUCTgrid_3x3min_nomask_c191114.nc"
```

Comment:
This works as written, but I would rather return to using line 299
`INGRID[nfile]=`$QUERYFIL``
and modify $QUERYFIL to pick up files according to their `frm_hgrid` rather than their `hgrid` (see [relevant post](https://github.com/ESCOMP/CTSM/pull/823/commits/a977c8a3f27a3054691abb234100e9840afeb81d#r347004693).

Could @billsacks or @ekluzek pls help me accomplish that?

Reply:
> Could @billsacks or @ekluzek pls help me accomplish that?

I think I've solved it. I'll keep you posted.